scripts/verify-bun-review-artifacts.py

```
@@ -13,7 +13,7 @@
 import re
 import sys
 import tempfile
-from typing import Any, Callable
+from typing import Any, Callable, NoReturn
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)